src/assets/Generator.Shared/VoidValue.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+
+namespace Azure.Core
+{
+    internal readonly struct VoidValue { }
```

Comment:
Our plan is to remove this once https://github.com/Azure/azure-sdk-for-net/pull/29061 is merged?

Reply:
Removed this to combine to Single VoidValue struct.